Vagrantfile

```diff
@@ -154,7 +154,8 @@ Vagrant.configure(2) do |config|
     ln -fns  /opt/OwlServer/web/api /srv/owl/
     cp -f /vagrant/scripts/api-settings.js /srv/owl/api/
     # install nodejs and npm
-    apt-get install -y nodejs npm
+    curl -sL https://deb.nodesource.com/setup_6.x |  bash -
+    sudo apt-get install -y nodejs
```

Comment:
I double checked that this installation script supports both Debian Jessie (on ulrike) and Debian Wheezy (nestor).